docs/pages/installation/minikube.md

```diff
@@ -78,6 +78,8 @@ Use Meshery's CLI to streamline your connection to your Minikube cluster. Config
 {% capture code_content %}$ mesheryctl system config minikube{% endcapture %}
 {% include code.html code=code_content %}
 
+Note: The kube config created by minikube is minified and starting meshery will fail with tls issues. Please run this command to flatten the kube config file: `kubectl config view --minify --flatten > ~/.kube/config-2 && rm .kube/config && mv .kube/config-2 .kube/config`
```

Comment:
```suggestion
Note: The kubeconfig created by minikube is minified and starting Meshery will fail with TLS issues (an invalid certificate). Please run this command to flatten the kubeconfig file: `kubectl config view --minify --flatten > ~/.kube/config-2 && rm .kube/config && mv .kube/config-2 .kube/config`
```

Reply:
@innocentrda `mesheryctl system config` is a custom-written command to perform this action for the user. Have you tried this command?

Also, we shouldn't tell the user to do this `&& mv .kube/config-2 .kube/config` without significant warning as to what affect this will have.